### PR TITLE
Change "StarkNet" to "Starknet"

### LINKED ISSUE #398: Change "StarkNet" to "Starknet"
Change the spelling of "StarkNet" from "StarkNet" to "Starknet" everywhere in the doc following the recommendations in [this SNIP](https://community.starknet.io/t/snip-naming-convention-starknet-not-starknet).

### PR DESCRIPTION
**Pull request by Hug0x0**
Changes the spelling of "StarkNet" from "StarkNet" to "Starknet" everywhere in the doc following the recommendations in [this SNIP](https://community.starknet.io/t/snip-naming-convention-starknet-not-starknet).

Closes #398 

## Checklist:

- Updated the docs (page)

### CODE REVIEW

File: page/docs/guide/run.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ usage: starknet-devnet [-h] [-v] [--host HOST] [--port PORT] [--load-path LOAD_P
                        [--timeout TIMEOUT] [--account-class ACCOUNT_CLASS] [--fork-network FORK_NETWORK] [--fork-block FORK_BLOCK]
                        [--chain-id CHAIN_ID] [--blocks-on-demand]
 
-Run a local instance of StarkNet Devnet
+Run a local instance of Starknet Devnet
```
Comment (FabijanC) [inline page/docs/guide/run.md:15]: This should also be reflected in `devnet_config.py`

**Review by FabijanC — CHANGES_REQUESTED** [review]
Please also apply the change in .py files.

File: page/docs/guide/run.md (hunk 1 above)
Comment (Hug0x0) [reply page/docs/guide/run.md:15]: Sure, No problem

File: test/contracts/cairo/always_fail.cairo [hunk 2]
```diff
@@ -1,4 +1,4 @@
-// Declare this file as a StarkNet contract and set the required
+// Declare this file as a Starknet contract and set the required
```
Comment (FabijanC) [inline test/contracts/cairo/always_fail.cairo:1]: Changing the content of smart contract files might change some expected addresses and hashes, so I'd leave them intact.

File: test/contracts/cairo/contract.cairo [hunk 3]
```diff
@@ -1,4 +1,4 @@
-// Declare this file as a StarkNet contract and set the required
+// Declare this file as a Starknet contract and set the required
```
Comment (FabijanC) [inline test/contracts/cairo/contract.cairo:1]: Changing the content of smart contract files might change some expected addresses and hashes, so I'd leave them intact.

File: test/contracts/solidity/L1L2.sol [hunk 4]
```diff
@@ -2,14 +2,14 @@
 pragma solidity ^0.6.12;
 
 /**
-  Demo contract for L1 <-> L2 interaction between an L2 StarkNet contract and this L1 solidity
+  Demo contract for L1 <-> L2 interaction between an L2 Starknet contract and this L1 solidity
```
Comment (FabijanC) [inline test/contracts/solidity/L1L2.sol:5]: Changing the content of smart contract files might change some expected addresses and hashes, so I'd leave them intact.

File: starknet_devnet/blueprints/rpc/rpc_spec.py [hunk 5]
```diff
@@ -7,7 +7,7 @@
     "openrpc": "1.0.0-rc1",
     "info": {
         "version": "0.45.0",
-        "title": "StarkNet Node API",
+        "title": "Starknet Node API",
```
Comment (FabijanC) [inline starknet_devnet/blueprints/rpc/rpc_spec.py:10]: @cptartur I feel like we haven't kept the specs copy-pasted so far, so this change could be acceptable? Or do we only want to update the specs when a new spec version is released?

**Review by FabijanC — CHANGES_REQUESTED** [review]

File: starknet_devnet/blueprints/rpc/rpc_spec.py (hunk 5 above)
Comment (cptartur) [reply starknet_devnet/blueprints/rpc/rpc_spec.py:10]: I think it's okay to change it 👍

File: test/contracts/cairo/always_fail.cairo (hunk 2 above)
Comment (Hug0x0) [reply test/contracts/cairo/always_fail.cairo]: Yes of course

**Review by FabijanC — APPROVED** [review]

**Pull request merged**